frontend/test/__support__/e2e/commands/api/composite/createQuestionAndAddToDashboard.js

```diff
@@ -1,10 +1,15 @@
 Cypress.Commands.add(
   "createQuestionAndAddToDashboard",
   (query, dashboardId) => {
-    return cy.createQuestion(query).then(response => {
-      cy.request("POST", `/api/dashboard/${dashboardId}/cards`, {
-        cardId: response.body.id,
-      });
+    return (query.native
+      ? cy.createNativeQuestion(query)
+      : cy.createQuestion(query)
+    ).then(response => {
```

Comment:
Doesn't[ `createQuestion` command](https://github.com/metabase/metabase/blob/master/frontend/test/__support__/e2e/commands/api/question.js) allow to create both GUI and native questions?

Reply:
@kulyk, unfortunately, it's not. Maybe we don't need two different commands 🤔

Reply:
@nemanjaglumac hmm, I got an error, maybe I misused it. Do we need different commands then?